marketplaceItems/sitPoint/appResources/appData/sitClient.js

```diff
@@ -814,95 +744,112 @@
 
     // #region ENTITY METHODS
 
-    // Constants
-    var ENCOUNTERED_CHAIR_CHECK_INTERVAL = 500; // ms
-
-    // Dynamic variables
-    var entityID = null;
-    var intervalEncounteredChairCheck = null;
-
-    function SitClient() {}
+    var ENCOUNTERED_CHAIR_CHECK_INTERVAL_MS = 500;
+    var that;
+    function SitClient() {
+        that = this;
+        this.entityID = null;
```

Comment:
you want these declarations to be inside your prototype instantiation's preload, not here. (see Slack)